spec/requests/cloud_volumes_spec.rb

```diff
@@ -218,6 +218,7 @@
   end
 
   describe 'create backup' do
+    let(:invalid_cloud_volume_url) { api_cloud_volume_url(nil, ApplicationRecord.id_in_region(999_999, ApplicationRecord.my_region_number)) }
```

Comment:
I think this should move into the clone context down here https://github.com/ManageIQ/manageiq-api/pull/1164/files#diff-a3c50450d3d70a77c2bddd76092b5ba015fba5015545be02eea66f7850ba52f1R327

Reply:
Oh weird I just noticed the attach/detach specs are in the 'create backup' describe block.  Can you move the clone volume tests into their own describe block and out of the 'create backup' one?

Reply:
Will do!